src/core/tracking.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-underscore-dangle */
+
+import config from 'config';
+import log from 'core/logger';
+
+
+export class Tracking {
+
+  constructor({trackingId, trackingEnabled, _log = log} = {}) {
+    if (typeof window === 'undefined') {
+      return;
+    }
+    this._log = _log;
+    this.id = trackingId;
+    this.enabled = trackingEnabled && trackingId;
+    this.logPrefix = `[GA: ${trackingEnabled ? 'ON' : 'OFF'}]`;
+
+    if (this.enabled) {
+      _log.info('Tracking init (Analytics is ON)');
+      /* eslint-disable */
+      // Snippet from Google UA docs: http://bit.ly/1O6Dsdh
+      window.ga = window.ga || function() {(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
+      ga('create', trackingId, 'auto');
+      ga('send', 'pageview');
+      /* eslint-enable */
+    } else {
+      _log.info('Tracking init. (Analytics is OFF)');
+      if (!this.id) {
```

Comment:
To be enabled you have to have a trackingId and set trackingEnabled to true. There's a separate warning logged for that too.